atlasdb-client/src/main/java/com/palantir/atlasdb/compress/CompressionUtils.java

```diff
@@ -22,7 +22,7 @@
 import com.palantir.atlasdb.table.description.ColumnValueDescription.Compression;
 import com.palantir.common.base.Throwables;
 
-public class CompressionUtils {
+public final class CompressionUtils {
     private CompressionUtils() {
         // empty
```

Comment:
I imagine there are going to be a lot of these, and rather low value, but would be nice to be consistent